### PR TITLE
Added and Updated macOS Dependency Files

### PR DESCRIPTION
**Pull request by manoj1749**
- This pr is solving the issue #15 
- other pubsec files are updated in the previous pr's, so only the macOS dependency files are left to update along with those Podfiles, which were being done in this pr.

### CODE REVIEW

**Comment by manoj1749** [comment]
Heyy @Techno-Disaster can you look into this pr👀

**Comment by Techno-Disaster** [comment]
Can you confirm if Linux and windows need any updates?

**Comment by manoj1749** [comment]
- Yeah, I have checked and for 1 package it should be added in the dependencies and I had added it.
- Other than that nothing is there to be updated about the dependecies.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,54 +1,54 @@
-name: ccxgui
```
Comment (Techno-Disaster) [inline pubspec.yaml:1]: hm what changes here?

Comment (manoj1749) [reply pubspec.yaml:1]: The change might have been overwritten when I had resolved conflicts, lemme check it once

**Comment by manoj1749** [comment]
Now, I have added the package which was overwritten when I am resolving the conflicts.

**Pull request merged**